examples/access-plugin-minimal/watcherjob.go

```diff
@@ -28,7 +28,16 @@ func (g *googleSheetsClient) HandleEvent(ctx context.Context, event types.Event)
 		return nil
 	}
 
-	r := event.Resource.(types.AccessRequest)
+	if _, ok := event.Resource.(types.WatchStatus); ok {
```

Comment:
should this be `types.WatchStatusV1` ?

Reply:
I do not have any good opinion here if it would be any better (or know what we prefer throughout codebase). I just remembered that we have `v1` for that type and pointed it out if it was deliberate or you missed it. 

That said, I usually prefer to have strong type over interface so to avoid magic, both while reading the code and while carrying on with other people's work.